deliverydb/db.py

```diff
@@ -3,19 +3,16 @@
 import sqlalchemy
 
 from deliverydb.model import Base, Scan
+import model.compliancedb
 
 
 class DeliveryDB:
     def __init__(
         self,
-        username: str,
-        password: str,
-        hostname: str,
-        port: int,
-        dialect: str = 'postgresql',
+        db_conn_url: str,
```

Comment:
that's more like it. how about moving the util-function we use to create this connection-url either to the `ccc.deliverydb` module, or (e.g. as a method) to the `model.deliverydb` - oh wait: ...compliancedb cfg class? then, the deliverydb-client would not need to have a direct dependency towards those.

on an additional note: I just noticed we use (in the module / package names) both deliverydb and compliancedb. we should use consistent naming (-> please rename compliancedb -> deliverydb)

Reply:
Will move to `ccc.deliverydb` for now and follow up on renaming the cfg class.